loki-3.yaml

```diff
@@ -30,6 +30,12 @@ pipeline:
       replaces: go.etcd.io/bbolt=go.etcd.io/bbolt@v1.3.6
       show-diff: true
 
+  # fix compilation with go-1.23
+  # https://github.com/grafana/pyroscope-go/releases/tag/godeltaprof%2Fv0.1.8
+  - uses: go/bump
```

Comment:
@smoser Note for commentary one can always also use "name: Fix compilation failure with go-1.23" instead of a comment. In case you want to be more poetic.